app/views/partials/trials-list.html

```diff
@@ -64,15 +64,17 @@
 
     <ol class="trial-list" id="trials-list">
       {% for trial in trials %}
-      {% if trial.isCompleted %}
-      {% set class = '' %}
-      {% if trial.isCompleted and not trial.isPublished %}
-      {% if trial.daysAfterCompletion <= 150 %}
-      {% set class = 'ongoing' %}
-      {% else %}
-      {% set class = 'no-results' %}
-      {% endif %}
-      {% endif %}
+        {% if trial.completionDate.isValid() %}
+        {% if trial.isCompleted %}
```

Comment:
It's less broken now, but the `if trial.completionDate.isValid()` is still on the same indentation level of `trial.isCompleted`, which is inside it.